src/main/java/au/com/centrumsystems/hudson/plugin/util/BuildUtil.java

```diff
@@ -88,8 +97,8 @@
      *            - The AbstractProject
      * @return - AbstractBuild's ParametersAction
      */
-    public static Action getAllBuildParametersAction(//
-            final AbstractBuild<?, ?> upstreamBuild, final AbstractProject<?, ?> downstreamProject) { //
+    public static Action getAllBuildParametersAction(final AbstractBuild<?, ?> upstreamBuild,
+                                                     final AbstractProject<?, ?> downstreamProject) {

```

Comment:
This was not part of my initial commit, but checkstyle gave me a warning about this being over 140 characters.